hub/powertoys/newplus.md

```diff
@@ -0,0 +1,43 @@
+---
+title: PowerToys New+ for Windows
+description: A tool that enables you to create files and folders from an easily personalized set of templates, directly from the File Explorer context menu.
+ms.date: 07/13/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, New+, New, NewPlus, Win]
+---
+
+# New+
+
+**PowerToys New+** enables you to create files and folders from an easily personalized set of templates, directly from the File Explorer context menu.
+
+## Getting started
+
+### How to enable New+
+
+To start using New+, enable New+ in the PowerToys settings.
+
+### How to create a new object using New+
+
+To create a new item within a folder, right-click on the background of the folder to bring up the context menu. From there, click on the “New+” option and then select the template you were looking for.
+
+### How to add or customize a new template to New+
+
+To create a new template, start by right-clicking the background of a folder. This will open a context menu where you can select the ‘New+’ option. From there, choose ‘Open templates’ to access the “Templates” folder. In this folder, you have the freedom to add, edit, or rename objects as per your needs. It’s important to note that the objects you add here will be displayed on the ‘New+’ menu in a sorted order, with folders always appearing first. This makes it easy for you to find and select your templates.
```

Comment:
```suggestion
To create a new template, start by right-clicking on the folder. This will open a context menu where you can select the ‘New+’ option. From there, choose ‘Open templates’ to access the “Templates” folder. In this folder, you have the freedom to add, edit, or rename objects as per your needs. It’s important to note that the objects you add here will be displayed on the ‘New+’ menu in a sorted order, with folders always appearing first. This provides the ability to find and select your templates.
```